code/tests/functional/backend_api/conftest.py

```diff
@@ -55,10 +55,15 @@ def setup_default_mocking(httpserver: HTTPServer, app_config: AppConfig):
         }
     )
 
+    httpserver.expect_oneshot_request(
+        "/indexes",
```

Comment:
Changed how we're checking for index creation compared to langchain. Instead of trying to get it and handling the not found exception, I'm querying for it instead.